qt_dotgraph/src/qt_dotgraph/pydotfactory.py

```diff
@@ -174,4 +175,4 @@ def create_dot(self, graph):
         if type(dot) != str:
             dot = dot.decode()
         # sadly pydot generates line wraps cutting between numbers
-        return dot.replace('\\\n', '')
+        return dot.replace('\\%s' % os.linesep, '')
```

Comment:
Should this replace all newline types with '' instead of just `os.linesep`. I.e, is there any reason to believe there could be '\n' types of newlines on windows?

Reply:
@brawner Do you mean doing the replacement like the below? That works too. I didn't observe the `pydot` generates `\\\n` from my local repro (Windows environment), but I cannot say it is never going to happen neither. I don't have strong preference here. We can do the both in all platforms.
```python
return dot.replace('\\\n', '').replace('\\\r\n','')
```

Reply:
Or:

```python
return dot.replace('\\%s' % os.linesep, '').replace('\\\n', '')
```

I'm not familiar with how dotgraph generates this stuff, but I've run into issues where libraries don't always do the Windows thing on Windows.

Reply:
> I'm not familiar with how dotgraph generates this stuff, but I've run into issues where libraries don't always do the Windows thing on Windows.

That's fair enough. I changed the code to the one you proposed.